lib/queue_bus/config.rb

```diff
@@ -55,6 +55,10 @@ def incoming_queue
       @incoming_queue ||= "bus_incoming"
     end
 
+    def worker_middleware
+      @worker_middleware ||= QueueBus::Middleware::Stack.new
```

Comment:
Real minor, but IMO I'd prefer to see this called `middleware_stack` vs `worker_middleware`, just to better represent the object it is and reduce confusion of it being an actual middleware piece.

Reply:
I liked having `worker_` on it so that it was clear that this was the stack for the worker itself. Would you like me to make it `worker_middleware_stack`?